views/layouts/authentication.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+
+<html class="layout-authentication">
+  <head>
+    <title>Google Scraper</title>
+    <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
+    <link rel="shortcut icon" href="/static/icons/favicon.ico" />
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1, shrink-to-fit=no"
+    />
+  </head>
+
+  <body>
+    <div class="app-header">{{ template "shared/header.html" . }}</div>
```

Comment:
If it makes, prefer using `header` for semantic reasons. Same below, prefer to use `footer`.